scripts/start.sh

```diff
@@ -3,4 +3,4 @@
 . scripts/_select_docker_compose.sh
 
 $DOCKER_COMPOSE --profile=manual_dev down orchestrator_postgres_db_dev
-$DOCKER_COMPOSE up --wait
```

Comment:
Will this change start the `orchestrator_postgres_db_dev` and `orchestrator_postgres_db` at the same time?